### PR TITLE
[IOTDB-2184]Update the code of Kafka example to use Session API and u…

### PR DESCRIPTION
**Pull request by Alibaba-HZY**
[IOTDB-2184]Update the code of Kafka example to use Session API and update kafka version from 0.8.2.0 to 2.8.0

### CODE REVIEW

File: example/kafka/src/main/java/org/apache/iotdb/kafka/ConsumerThread.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        case FLOAT:
+          values.add(Float.parseFloat(valuesStr[i]));
+          break;
+        case BOOLEAN:
+          values.add(Boolean.parseBoolean(valuesStr[i]));
+          break;
+      }
+    }
+
+    session.insertRecord(device, time, measurements, types, values);
```
Comment (wangchao316) [inline example/kafka/src/main/java/org/apache/iotdb/kafka/ConsumerThread.java:84]: You are advised to use the session API and insertRecords, or insertTablets to insert data.This allows you to get the most out of write performance

File: example/kafka/src/main/java/org/apache/iotdb/kafka/ConsumerThread.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                Constant.IOTDB_CONNECTION_PORT,
+                Constant.IOTDB_CONNECTION_USER,
+                Constant.IOTDB_CONNECTION_PASSWORD);
+        session.open();
+        for (ConsumerRecord<String, String> record : records) {
+          insert(session, record.value());
+        }
+      } while (true);
+    } catch (Exception e) {
+      logger.error(e.getMessage());
```
Comment (wangchao316) [inline example/kafka/src/main/java/org/apache/iotdb/kafka/ConsumerThread.java:104]: When is the session closed?
If an exception occurs, do you want to close it?

**Review by wangchao316 — CHANGES_REQUESTED** [review]

File: example/kafka/src/main/java/org/apache/iotdb/kafka/ConsumerThread.java [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+    session.insertRecord(device, time, measurements, types, values);
+  }
+
+  @Override
+  public void run() {
+    try {
+      do {
+        ConsumerRecords<String, String> records = consumer.poll(Duration.ofSeconds(1));
+        Session session =
```
Comment (wangchao316) [inline example/kafka/src/main/java/org/apache/iotdb/kafka/ConsumerThread.java:92]: A new session is created in each loop. You can use the SessionPool.

**Review by wangchao316 — CHANGES_REQUESTED** [review]

File: example/kafka/src/main/java/org/apache/iotdb/kafka/ConsumerThread.java (hunk 3 above)
Comment (Alibaba-HZY) [reply example/kafka/src/main/java/org/apache/iotdb/kafka/ConsumerThread.java:92]: ok，i will use SessionPool

File: example/kafka/src/main/java/org/apache/iotdb/kafka/ConsumerThread.java (hunk 2 above)
Comment (Alibaba-HZY) [reply example/kafka/src/main/java/org/apache/iotdb/kafka/ConsumerThread.java:104]: yes i need to close it

File: example/kafka/src/main/java/org/apache/iotdb/kafka/Consumer.java [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+import org.apache.iotdb.tsfile.file.metadata.enums.TSDataType;
+import org.apache.iotdb.tsfile.file.metadata.enums.TSEncoding;
+
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
```
[the remark this answers is not in the document]
Comment (Alibaba-HZY) [reply example/kafka/src/main/java/org/apache/iotdb/kafka/Consumer.java:34]: i will repair it

**Review by wangchao316 — APPROVED** [review]
LGTM

**Review by HTHou — APPROVED** [review]

**Review by SteveYurongSu — APPROVED** [review]

**Pull request merged**